src/main/java/com/biku89/medical_clinic/InstitutionDTO.java

```diff
@@ -0,0 +1,4 @@
+package com.biku89.medical_clinic;
+
+public record InstitutionDTO(Long id,String name, String city, String postalCode, String street, String buildingNumber) {
```

Comment:
teraz instytucja posiada rowniez informacje o przypisanych do niej doktorach warto to zwrocic uzytkownikowi